draft-ietf-tls-tls13.md

```diff
@@ -1567,8 +1567,8 @@ cipher_suites
   the server does not recognize, support, or wish to use, the server
   MUST ignore those cipher suites, and process the remaining ones as
   usual. Values are defined in {{cipher-suites}}. If the client is
-  attempting a PSK key establishment, it MUST only advertise the single
-  cipher suite associated with the PSK.
+  attempting a PSK key establishment, it MUST advertise at least one
+  cipher suite containing a Hash associated with the PSK.
```

Comment:
Hrm. Perhaps there should be some text in 4.2.6 then, as guidance when picking parameters? If you're doing out-of-band PSKs where there isn't a cert-based default, you actually to not just say "please include some ciphers that match the PSK" but that "the only ciphers that will work are ones that match the PSK, so avoid saying silly things like AES_256_GCM_SHA384 when you don't have a cert-based default and all your PSKs are SHA-256 ones".